Library/Formula/libcutl.rb

```diff
@@ -0,0 +1,24 @@
+class Libcutl < Formula
+  homepage "http://www.codesynthesis.com/projects/libcutl/"
+  url "http://www.codesynthesis.com/download/libcutl/1.9/libcutl-1.9.0.tar.gz"
+  sha256 "1b575aa8ed74aa36adc0f755ae9859c6e48166a60779a5564dd21b8cb05afb7d"
+
+  depends_on "gcc" => :build
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  fails_with :clang do
+    cause <<-EOS.undent
+    Oh! It seems that you only have clang available, or GCC wasn't found!
+    Make sure you GCC is installed and recognized by homebrew.
+    EOS
+  end
+
+  test do
+    system "false"
```

Comment:
Thanks for the clarification. Was just curious as to whether, despite the GPLv2, there was some existing license complication that may not have played so well with Brew's FOSS-love.

> Concerning the tests, I get it. But as I don't "make" the archives, am I allowed to write the test files 

You can write temporary files to the testpath. You can see an example of that [here](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/sshrc.rb#L12-L20).